site/guide/model-documentation/work-with-content-blocks.qmd

```diff
@@ -3,27 +3,57 @@ title: "Work with content blocks"
 date: last-modified
 ---
 
-Make edits to your model documentation or validation reports by adding or removing content blocks directly in the online editor. 
+Make edits to your model documentation, validation reports, or ongoing monitoring plans by adding or removing content blocks directly in the online editor. 
 
 ## What are content blocks? 
 
-Content blocks provide you with sections that are part of a template. You can think of these sections as an empty canvas that you fill in with text and test results. Multiple sections are joined to create a longer document with a table of contents that has different heading and subheading levels, such as "1.," "1.1.," and so on. 
+Content blocks provide you with sections that are part of a template. 
 
-Types of content blocks: 
+- You can think of these sections as an empty canvas that you fill in with text and test results. 
+- Multiple sections are joined to create a longer document with a table of contents that has different heading and subheading levels, such as "1.," "1.1.," and so on. 
 
-Simple text block
-: Can be added anywhere on model documentation or validation reports and edited to include additional documentation in text format.
+#### Content block types
+
+simple text block
+: Can be added anywhere on model documentation, validation reports, or ongoing monitoring plans and edited to include additional documentation in text format.
+
+test-driven block
+: Can be within model documentation and ongoing monitoring plans to display one of the supported test results collected by the {{< var validmind.developer >}}.
+
+metric over time block
```

Comment:
Nothing to do here but it looks like we hyphenated the noun phrase "test-driven" in the product but then didn't hyphenate "metric-over-time" to remove ambiguity.

Reply:
Yeah, that's how they look in the UI. I don't know if that's just how those standard terms work, you'd have to ask @johnwalz97 perhaps?